refine.bat

```diff
@@ -171,7 +171,6 @@ set REFINE_LIB_DIR=server\target\lib
 :gotLibDir
 
 if not "%REFINE_VERBOSITY%" == "" goto gotVerbosity
-set REFINE_VERBOSITY=info
```

Comment:
I guess we probably need to keep this, no?

Reply:
If we keep it, and an argument is not supplied in /v, it would default to INFO.
This would make it necessary to supply an argument and would never behave as per the configuration specified in log4j2.properties file.